src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpGoToDefinitionNet60.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Structure;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Microsoft.VisualStudio.Shell.TableControl;
+using Roslyn.VisualStudio.IntegrationTests;
+using Roslyn.VisualStudio.IntegrationTests.InProcess;
+using Xunit;
+
+namespace Roslyn.VisualStudio.NewIntegrationTests.CSharp
+{
+    [Trait(Traits.Feature, Traits.Features.GoToDefinition)]
+    public class CSharpGoToDefinitionNet60 : AbstractEditorTest
+    {
+        protected override string LanguageName => LanguageNames.CSharp;
+
+        public CSharpGoToDefinitionNet60()
+            : base(nameof(CSharpGoToDefinition), WellKnownProjectTemplates.CSharpNetCoreClassLibrary)
+        {
+        }
+
+        [IdeFact]
+        public async Task ConsoleWriteLine()
```

Comment:
Disclaimer: I have no idea if this test will pass in CI, because this feature is very highly dependent on what is installed on the CI machine...

Reply:
I think it passed, but with the infra as it is, its hard to tell :D